manifest.json

```diff
@@ -7,5 +7,39 @@
     "matches": ["app://system.gaiamobile.org/index.html"],
     "js": ["addon.js"]
   }],
-  "web_accessible_resources": ["assets/Home.png", "assets/Back.png", "index.html"]
+  "web_accessible_resources": [
+    "node_modules/fxos-icons/fxos-icons.css",
+    "node_modules/fxos-icons/fonts/fxos-icons.ttf",
+    "node_modules/dom-scheduler/dom-scheduler.js",
+    "node_modules/fxos-component/fxos-component.js",
+    "apps/homescreen/index.html",
+    "apps/homescreen/image.png",
+    "apps/bbc.com/index.html",
+    "apps/bbc.com/image.png",
+    "apps/camera/index.html",
+    "apps/camera/image.png",
+    "apps/huffingtonpost.com/index.html",
+    "apps/huffingtonpost.com/image.png",
+    "apps/mashcreative.com/index.html",
+    "apps/mashcreative.com/image.png",
+    "apps/nytimes.com/index.html",
+    "apps/nytimes.com/image.png",
+    "apps/squarespace.com/index.html",
+    "apps/squarespace.com/image.png",
+    "apps/teamliquid.com/index.html",
+    "apps/teamliquid.com/image.png",
+    "apps/theverge.com/index.html",
+    "apps/theverge.com/image.png",
+    "apps/vine.com/index.html",
+    "apps/vine.com/image.png",
+    "apps/wired.com/index.html",
+    "apps/wired.com/image.png",
+    "assets/urlbar_refresh_icon.png",
+    "assets/urlbar_reader_mode_icon.png",
+    "lib/app.css",
+    "lib/app.js",
+    "lib/hope-tab.js",
+    "lib/hope-tab-list.js",
+    "index.html"
```

Comment:
There is already [a bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1217944) for this.